### ISSUE TITLE
File history is not working

### ISSUE THREAD

**Issue by antonioprates** [Murmele/Gittyup#565]
One core feature seems to be broken, can't see the file history as I can see on Git Ahead:

This is GitAhead:
![git-ahead-Screenshot_2023-05-26_09-53-29](https://github.com/Murmele/Gittyup/assets/26195023/844ed323-1f06-4a73-bbfe-021c2fbb3f5d)

And this is Gittyup:
![gittyup-Screenshot_2023-05-26_09-56-40](https://github.com/Murmele/Gittyup/assets/26195023/fff0548d-466a-4cb9-abaa-1f9cdc522b7f)

I don't think it's related to my setup, but here are my system configurations:

OS	 Manjaro Linux x86_64  
Kernel	 6.1.29-1-MANJARO 
DE	 Xfce 4.18 
WM	 Xfwm4

**Comment by antonioprates** [comment]
I used pacman, give me a minute I can try flatpak.

**Comment by antonioprates** [comment]
Right, it works with flatpak, not favorite choice for installing apps, e.g. this added almost up about 1GB in flatapak dependencies, but at least it works. Maybe it relies on some outdated lib, that could be why it does not work from AUR package, not sure, as I have not looked into the code.

**Comment by Murmele** [comment]
So as conclusion:

- Flatpak: works fine
- Pacman: Does not work
- AUR: ?

**Comment by antonioprates** [comment]
As far as I understand there is no binary version, only an AUR package is available to install. The actual command I have used was:

```
pamac install gittyup
```

Pamac builds the PKG and then uses the pacman backend (ALPM) to install the package build from AUR. So I think installing using pacman and AUR mean the same thing in this context.

**Comment by antonioprates** [comment]
TL;DR. Flatpak works, pacman/AUR no

**Comment by Murmele** [comment]
Did you try gittyup-git from aur?

**Comment by antonioprates** [comment]
That other AUR package won't complete build:

```
$ pamac install gittyup-git

...

92% tests passed, 2 tests failed out of 26

Label Time Summary:
NO_WIN32_OFFSCREEN    =   5.56 sec*proc (4 tests)

Total Test time (real) =  39.31 sec

The following tests FAILED:
	  7 - index (Failed)
	 17 - TreeView (Failed)
Errors while running CTest
```

**Comment by Murmele** [comment]
@alerque Do you know anything here?
